### ISSUE TITLE
Issues with Pt locale

### ISSUE THREAD

**Issue by smartinsightsfromdata** [nicolaskruchten/pivottable#410]
Users have reported problems with the  Pt locale (C3 and D3 renderers text is not translated).

I've noticed that the code for the Pt locale is still referring to "old" names like `Line Chart C3` : is it possible that this is the cause for the failed translation?

This is the code for the Pt locale:
```
    if (d3r) {
      $.pivotUtilities.locales.pt.d3_renderers = {
        "Mapa de Árvore": d3r["Treemap"]
      };
    }
    if (c3r) {
      $.pivotUtilities.locales.pt.c3_renderers = {
        "Gr&aacute;fico de Linhas": c3r["Line Chart C3"],
        "Gr&aacute;fico de Barras": c3r["Bar Chart C3"]
      };
    }
````

**Comment by nicolaskruchten** [comment]
Thanks for reporting this. I'll take a look this weekend as I am preparing
another PivotTable.js release.

Nicolas

**Comment by nicolaskruchten** [comment]
Should be resolved with [Version 2.0.0](https://github.com/nicolaskruchten/pivottable/releases/tag/v2.0.0)

Please note that a few things have changed with this version, so while I strongly recommend that you upgrade rpivotTable to using it, I encourage you to carefully consider the impact of these changes. I'd be happy to help you if they cause any issues for you.

BTW, because I don't have your email address (feel free to send me an email at nicolas@kruchten.com so that I do for the future!), I wanted to let you know that I presented your work to the Montreal R User group last week, and they were very impressed! Here's the blog post: http://nicolas.kruchten.com/content/2015/11/r-montreal/

I've also made it possible for Python users to have access to this library, inspired by your R integration: http://nicolas.kruchten.com/content/2015/09/jupyter_pivottablejs/
